spec/requests/api/v1/targets/show_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe 'Api::V1::Targets', type: :request do
+  describe 'GET #show' do
+    context 'success' do
+      before { get api_v1_target_url(target.id), as: :json }
```

Comment:
You can do 
`subject { get api_v1_target_url(target_id), as: :json } `

Then, in the success context do 
`let(:target_id) { target.id } `

In the error context do
`let(:target_id) {  'invalid_id'  } `